src/pages/StableSwap/index.tsx

```diff
@@ -462,59 +486,59 @@ export default function StableSwap() {
                     </ButtonConfirmed>
                     <ButtonError
                       onClick={() => {
-                        // @nocommit -- this removes the confirmation modal
-                        // if (isExpertMode) {
-                        handleSwap()
-                        // } else {
-                        //   setSwapState({
-                        //     tradeToConfirm: trade,
-                        //     attemptingTxn: false,
-                        //     swapErrorMessage: undefined,
-                        //     showConfirm: true,
-                        //     txHash: undefined
-                        //   })
-                        // }
+                        if (isExpertMode) {
+                          handleSwap()
+                        } else {
+                          setSwapState({
+                            tradeToConfirm: trade,
+                            attemptingTxn: false,
+                            swapErrorMessage: undefined,
+                            showConfirm: true,
+                            txHash: undefined
+                          })
+                        }
                       }}
                       width="48%"
                       id="swap-button"
                       disabled={
-                        !isValid || approval !== ApprovalState.APPROVED || (priceImpactSeverity > 3 && !isExpertMode)
+                        !isValid ||
+                        approval !== ApprovalState.APPROVED ||
+                        (priceImpactSeverity === true && !isExpertMode)
                       }
-                      error={isValid && priceImpactSeverity > 2}
+                      error={isValid && priceImpactSeverity === true}
```

Comment:
Would this show an error and not be disabled?

Reply:
It would show a Swap Anyways error colour which is not disabled